lighthouse-cli/chrome-finder.ts

```diff
@@ -200,11 +171,11 @@ function canAccess(file: string): Boolean {
 }
 
 function uniq(arr: Array<any>) {
-    return Array.from(new Set(arr));
+  return Array.from(new Set(arr));
 }
 
 function findChromeExecutables(folder: string): Array<string> {
-  const argumentsRegex = /(^[^ ]+).*/; // Take everything up to the first space
+  const argumentsRegex = /(^[^ ]+).*/;  // Take everything up to the first space
```

Comment:
SpacesBeforeTrailingComments

is the ticket changing to 1 to align with lighthouse.. although this is against google style guide norms